<launch>
  <!--node cwd="node" name="rosbag_node" pkg="rosbag" type="record"
  output="screen" args="-o $(find aauship_control)/bags /kf_states /imu /gps1 /lli_input"/-->

  <node cwd="node" name="model_node" pkg="aauship_control" type="model-node.py" output="screen"/>
  <node cwd="node" name="lqr_node" pkg="aauship_control" type="lqr_node" output="screen"/> 
</launch>